packages/@aws-cdk/cdk/lib/aspects/README.md

```diff
@@ -0,0 +1,345 @@
+## Tagging with the CDK
```

Comment:
This is not the correct place for this README as it won't be surfaced in our docs this way. Can you move this to the @aws-cdk/cdk README? I know it's pretty empty right now, but basically that's where it should go. Simply append this README to that one.

Reply:
Sure. The major change from this README to the guide will be that the guide does not discuss implementation. The purpose of a guide is to answer the questions:

- What is this feature?
- Why should I care?
- How do I use it?